src/react/services/api.js

```diff
@@ -156,6 +157,24 @@ export function deleteEntry(id, config, nonce = false) {
   return secureAjax(settings);
 }
 
+export function lockEntry(entryId, lock, config, nonce = false) {
+  const settings = {
+    method: 'POST',
+    body: JSON.stringify({
+      post_id: config.post_id,
+      entry_id: entryId,
```

Comment:
When do we use camel case (`entryId`) and when do we use snake_case (`entry_id`)?

Reply:
Camel case is for JavasScript variables and snake_case is for WordPress.